codelabs/gRPC_Cpp_OpenTelemetry_Plugin/protos/helloworld.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 gRPC authors.
```

Comment:
```suggestion
// Copyright 2024 gRPC authors.
```

Reply:
Wow CPP repo is very old

Reply:
no, we're not supposed to change the years since i'm reusing content